use-location.js

```diff
@@ -20,7 +20,7 @@ export default ({ base = "" } = {}) => {
     // unfortunately, we can't rely on `path` value here, since it can be stale,
     // that's why we store the last pathname in a ref.
     const checkForUpdates = () => {
-      const pathname = currentPathname(base);
```

Comment:
Can we get the old name back, please 🙏 
`currentPathname` is just more readable

Reply:
Of course, I just want to reduce the number of bytes